contrib/completion/bash/docker

```diff
@@ -201,19 +201,36 @@ __docker_signals() {
 
 _docker_docker() {
 	local boolean_options="
-		--daemon -d
 		--debug -D
 		--help -h
+		--tls
+		--tlsverify
+		--tlscacert
+		--tlscert
+		--tlskey
```

Comment:
See the completion for `--pidfile|-p|--tlscacert|--tlscert|--tlskey)` in `_docker_daemon()`.
By the way. if you would prefer to concentrate on real coding stuff, I could take over the bash completion part. Just give me a ping.